src/uucore/src/lib/features/format/num_format.rs

```diff
@@ -141,7 +141,17 @@ impl Formatter for UnsignedInt {
         let mut s = match self.variant {
             UnsignedIntVariant::Decimal => format!("{x}"),
             UnsignedIntVariant::Octal(Prefix::No) => format!("{x:o}"),
-            UnsignedIntVariant::Octal(Prefix::Yes) => format!("{x:#o}"),
+            UnsignedIntVariant::Octal(Prefix::Yes) => {
+                // The prefix that rust uses is `0o`, but GNU uses `0`.
+                // We also need to take into account that 0 is should not be 00
```

Comment:
There are either words missing after "is" or it can be removed.

Reply:
Ah thanks!